crates/sage-cli/src/main.rs

```diff
@@ -89,6 +103,105 @@ impl Runner {
         })
     }
 
+    pub fn prefilter_peptides(self, parallel: usize, fasta: Fasta) -> Vec<Peptide> {
```

Comment:
`prefilter` might not be specific enough. Would it make sense to rename it to something like `search_chunked_peptides` or similar?